modular_zubbers/code/game/gamemodes/fleshmind/fleshmind_mobs.dm

```diff
@@ -0,0 +1,1504 @@
+		BB_EMOTE_SOUND = list(
+			'modular_zubbers/sound/fleshmind/hiborg/passive_01.ogg',
+			'modular_zubbers/sound/fleshmind/hiborg/passive_02.ogg',
+			'modular_zubbers/sound/fleshmind/hiborg/passive_03.ogg',
+			'modular_zubbers/sound/fleshmind/hiborg/passive_04.ogg',
+			'modular_zubbers/sound/fleshmind/hiborg/passive_05.ogg',
+		)
+	)
+	/// The chance of performing a stun attack.
+	var/stun_attack_prob = 30
+	/// The chance of performing an AOE attack.
+	var/aoe_attack_prob = 15
+	/// The range on our AOE attaack
+	var/aoe_attack_range = 1
+	/// How often the mob can use the stun attack.
+	var/stun_attack_cooldown = 15 SECONDS
+	COOLDOWN_DECLARE(stun_attack)
+
+/mob/living/basic/fleshmind/hiborg/Initialize(mapload)
+	. = ..()
+	var/datum/action/cooldown/hiborg_slash/new_action = new
+	new_action.Grant(src)
+	var/list/loot = string_list(list(/obj/effect/gibspawner/robot))
+	AddElement(/datum/element/death_drops, loot)
+
+/mob/living/basic/fleshmind/hiborg/melee_attack(atom/target, list/modifiers, ignore_cooldown = FALSE)
+	. = ..()
+	if(prob(stun_attack_prob) && !key)
+		stun_attack(target)
+	if(prob(aoe_attack_prob) && !key)
+		aoe_attack()
+
+/mob/living/basic/fleshmind/hiborg/proc/stun_attack(mob/living/target_mob)
+	if(!COOLDOWN_FINISHED(src, stun_attack))
+		return
+	if(!ishuman(target_mob))
+		return
+	var/mob/living/carbon/human/attacked_human = target_mob
+	attacked_human.Paralyze(10)
+	playsound(src, 'sound/weapons/egloves.ogg', 50, TRUE)
+
+	COOLDOWN_START(src, stun_attack, stun_attack_cooldown)
+
+/mob/living/basic/fleshmind/hiborg/proc/aoe_attack()
+	visible_message("[src] spins around violently!")
+	spin(20, 1)
+	for(var/mob/living/iterating_mob in view(aoe_attack_range, src))
+		if(iterating_mob == src)
+			continue
+		if(faction_check(faction, iterating_mob.faction))
+			continue
+		playsound(iterating_mob, 'sound/weapons/whip.ogg', 70, TRUE)
+		new /obj/effect/temp_visual/kinetic_blast(get_turf(iterating_mob))
+
+		var/atom/throw_target = get_edge_target_turf(iterating_mob, get_dir(src, get_step_away(iterating_mob, src)))
+		iterating_mob.throw_at(throw_target, 20, 2)
+
+/datum/action/cooldown/hiborg_slash
+	name = "Slash (AOE)"
+	desc = "Whip everyone in a range."
+	button_icon = 'icons/obj/weapons/grenade.dmi'
+	button_icon_state = "slimebang_active"
+	cooldown_time = 20 SECONDS
+
+/datum/action/cooldown/hiborg_slash/Activate(atom/target)
+	if(!istype(owner, /mob/living/basic/fleshmind/hiborg))
+		return
+	var/mob/living/basic/fleshmind/hiborg/hiborg_owner = owner
+	hiborg_owner.aoe_attack()
+	StartCooldownSelf()
+
+/**
+ * Mauler
+ *
+ * Special ability: Tears chunks out of things.
+ * Malfunction chance: HIGH
+ *
+ * The mauler is a converted monkey, it's a mad ape!
+ *
+ * Created by converted monkeys.
+ */
+/mob/living/carbon/human/species/monkey/angry/mauler
+
+/**
+ * Himan
+ *
+ * Special ability: Shriek that stuns, the ability to play dead.
+ *
+ * Created by converted humans.
+ */
+
+/mob/living/basic/fleshmind/himan
```

Comment:
250 health mob with 25-30 damage attacks, with a 5 seconds stun AoE attack is crazy.

Reply:
It fakes its death at half health, but I've reduced the scream AOE